source/_integrations/vesync.markdown

```diff
@@ -99,6 +109,33 @@ VeSync air purifiers will expose the following details depending on the features
 | `night_light`           | The current status of the night light (Core200S)                                  | off             |
 | `child_lock`            | The current status of the child lock (Core200S)                                   | off             |
 
+## Humidifier Exposed Attributes
+
+VeSync air purifiers will expose the following details depending on the features supported by the model:
+
+| Attribute                     | Description                                                                       | Example             |
+| ----------------------------- | --------------------------------------------------------------------------------- | ------------------- |
+| `humidity`                    | The target humidity of the humidifier.                                            | 40                  |
```

Comment:
I think we should avoid using tables, as they are hard to read on mobile devices.

Reply:
Is avoiding tables more important than consistency on the same page? The docs for all other devices on this page use a table of this form for exposed attributes.

Reply:
> Is avoiding tables more important than consistency on the same page?

Make it consistent without table than 🤷

Reply:
I'm confused. Are you suggesting I convert all existing tables in this file into some other form? That's the only way I could avoid a table here *and* make it consistent with the rest of the page.

Reply:
>  Are you suggesting I convert all existing tables in this file into some other form?

Yes, that is what I was suggesting.

Reply:
IMO that’s out of the scope of my feature addition, but I’ll convert the tables in this PR into some other form so I’m at least not creating more technical debt.

Reply:
I went ahead and converted the new tables to lists, and also replaced the entity IDs with generic names. Lmk if there's anything else that needs to be changed.